cpp/dolfinx/geometry/BoundingBoxTree.cpp

```diff
@@ -20,7 +20,8 @@ namespace
 //-----------------------------------------------------------------------------
 // Compute bounding box of mesh entity
 Eigen::Array<double, 2, 3, Eigen::RowMajor>
-compute_bbox_of_entity(const mesh::Mesh& mesh, int dim, std::int32_t index)
+compute_bbox_of_entity(const mesh::Mesh& mesh, int dim, std::int32_t index,
+                       double padding)
```

Comment:
Do we need (or want?) to add padding here? I think it is better to just say "compute_bbox_of_entity" returns the bbox of the entity, and then you can pad it later, if you want to.